src/jetstream/plugins/desktop/helm/tokens.go

```diff
@@ -0,0 +1,60 @@
+package helm
+
+import (
+	"github.com/cloudfoundry-incubator/stratos/src/jetstream/repository/interfaces"
```

Comment:
same problema s above, if it's not a problem within the fork if we reference the archived repo, ingore my comment :)

Reply:
Yeah, this is indeed problematic and probably will not compile. If it does compile, then it will pull an undesired dependency on some older, published version of respository/interfaces (those do unfortunately exist). 
This is why, as soon as is feasible, I'd like to modify the entire jetstream package to have a new module path (probably something like `github.com/anynines/stratos/jetstream`).
I will put in an additional commit now to fix this manually in both places, but long term we need an un-ambiguous module path for our fork 👍🏻 Good spotting!